### PR TITLE
x11-drivers/nvidia-drivers: updates to 535.104.05

### PR DESCRIPTION
**Pull request by sayanchowdhury**
# x11-drivers/nvidia-drivers: updates to 535.104.05

This also fixes the broken nvidia.service as reported in https://github.com/flatcar/Flatcar/issues/1169 

CI for the nvidia upgrade has passed: http://jenkins.infra.kinvolk.io:8080/job/container/job/packages_all_arches/2458/

CI for the broken nvidia.service running:
http://jenkins.infra.kinvolk.io:8080/job/container/job/packages_all_arches/2467/cldsv/

- [x] Changelog entries added in the respective `changelog/` directory (user-facing change, bug fix, security fix, update)
- [x] Inspected CI output for image differences: `/boot` and `/usr` size, packages, list files for any missing binaries, kernel modules, config files, kernel modules, etc.

### CODE REVIEW

**Comment by sayanchowdhury** [comment]
CI passed http://jenkins.infra.kinvolk.io:8080/job/container/job/test/14075/cldsv/

File: changelog/security/2023-09-08-nvidia-drivers-535.104.05.md [hunk 1]
```diff
@@ -0,0 +1 @@
+- grub ([CVE-2023-25515](https://nvd.nist.gov/vuln/detail/CVE-2023-25515), [CVE-2023-25516](https://nvd.nist.gov/vuln/detail/CVE-2023-25515))
```
Comment (dongsupark) [inline changelog/security/2023-09-08-nvidia-drivers-535.104.05.md:1]: `s/grub/nvidia-drivers/` ?

Comment (dongsupark) [reply changelog/security/2023-09-08-nvidia-drivers-535.104.05.md:1]: The second CVE should be like `[CVE-2023-25516](https://nvd.nist.gov/vuln/detail/CVE-2023-25516)`.

**Comment by dongsupark** [comment]
Changelog looks good.

[CI test](http://jenkins.infra.kinvolk.io:8080/job/container/job/test/14075/) seems to still fail.

**Comment by dongsupark** [comment]
CI fails http://jenkins.infra.kinvolk.io:8080/job/container/job/test/14306/

**Review by dongsupark — APPROVED** [review]
Let's merge for now, and fix the failing CI test later.

**Pull request merged**